abs.yaml

```diff
@@ -1 +1,3 @@
 aggregate_branch: 3.12
```

Comment:
```suggestion
```
this can be removed now, was historical (for building python 3.12 packages before python v3.12 was added to the build matrix formally)